README.md

```diff
@@ -330,753 +277,454 @@ Rails.application.routes.draw do
 end
 ```
 
-Now, we need a view! This will be similar, but not exactly the same as the view we created for the `new` action. We will need to create the following HTML in `app/views/tasks/edit.html.erb`
+Another new method! We have used new, save, find, and update so far - what is delete doing? delete is another method we are inheriting from ActiveRecord that deletes records in our database based on an id that you send to the method.
 
-**app/views/tasks/edit.html.erb**
+Now, we should be able to send a `DELETE` request to http://localhost:3000/tasks/:id and delete the task for the provided id!
 
-```html
-<form action="/tasks/<%= @task.id %>" method="post">
-  <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>">
-  <input type="hidden" name="_method" value="PATCH" />
-  <p>Edit</p>
-  <input type='text' name='title' value="<%= @task.title %>"/><br/>
-  <textarea name='description'><%= @task.description %></textarea><br/>
-  <input type='submit'/>
-</form>
-```
-
-The way that we have set up this form, including the `@task.title` and `@task.description` as field values, will autofill our form with the current task information, so that a user can update one or both fields and maintain any unchanged information.
-
-Additionally, you'll notice that there's a hidden field with a value of PATCH. So far in our Routes, we have used GET requests to retrieve data and POST requests to create data. Now we are going to use a PATCH request to update data. Normally, HTML forms only allow GET or POST requests (see more information [here](http://www.w3schools.com/tags/att_form_method.asp)). HTML won't allow us to use method='patch' in our form tag, but passing it as a hidden value gives our app the information it needs to route the request correctly.
 
-Why do we want this to be a PATCH request? In a word, convention. Take a quick look at [this](https://www.restapitutorial.com/lessons/httpmethods.html).
+### One Step Further - Serializers
 
-### Updating a Task Resource
+The goal for this tutorial is to walk you through the full process of building an API that supports the fully CRUD actions. We have successfully done that and we could stop here!
 
-Now, our application will need to know what to do when someone submits the edit form, making a `PATCH` request to `/tasks/:id`; so, in our `config/routes.rb` let's add the following route:
+But often times we will want to customize our response a little bit more instead of sending back the full object. For us to accomplish this, we are going to use something called a Serializer.
 
-**config/routes.rb**
-
-```ruby
-patch '/tasks/:id', to: 'tasks#update'
+```
+$ mkdir -p app/serializers
+$ touch app/serializers/task_serializer.rb
 ```
 
-And in our tasks controller, let's add an `update` action:
+<!-- TODO, any ideas for something more real world to do with this serializer besides not return the description? -->
```

Comment:
I would probably ignore either (or both) the `created_at` or `updated_at`. Yes there are times where that's valuable to have but of all the attributes those are most likely to NOT be rendered on the FE and the format is hard to read without parsing.

Reply:
Agree, we should exclude the timestamps, maybe even the primary key?

Reply:
Love that idea to remove the timestamps.